README.md

```diff
@@ -85,29 +170,29 @@ spec:
     namespace: default
 ```
 
-NHC creates FAR CR using FAR Template after it detects an unhealthy node (according to NHC unhealthy conditions).
-FAR CRs are deleted by NHC after it sees the Node is healthy again.
+NHC creates FenceAgentsRemediation CR using fartemplate after it detects an unhealthy node (according to NHC's unhealthy conditions).
+FenceAgentsRemediation CRs are deleted by NHC after it detects the node is healthy again.
 
 ### Standalone FAR
 
 * Install FAR using one of the above options ([Installation](#installation)).
 
-* Create FAR CR using the name of the node to be remediated, and the fence-agent parameters.
+* Create FenceAgentsRemediation CR with the name of the node to be remediated, the fence agent name, and its parameters.
 
-#### Example CR
-
-The FAR CR (CustomResource), created by the admin, is used to trigger the fence-agent on a specific node.
+#### Example FenceAgentsRemediation CR
 
+The FAR CR, `FenceAgentsRemediation`, is created by the admin and is usedto trigger the fence agent on a specific node.
 The CR includes the following parameters:
 
-* `agent` - fence-agent name
-* `sharedparameters` - cluster wide parameters for executing the fence agent
-* `nodeparameters` - node specific parameters for executing the fence agent
-* `retrycount` - number of times to retry the fence-agent in case of failure. The default is 5.
+* `agent` - fence agent name. The name is validated (by kuebuilder and Webhook) against a list of supported agents in the FAR pod.
```

Comment:
typo: kuebuilder -> kubebuilder

Reply:
It might also be useful explicit that by fence_agent name we actually intend the name of the executable file